### ISSUE TITLE
Possible mistake in alpha loss in SAC

### ISSUE THREAD

**Issue by perrin-isir** [aai-institute/tianshou#557]
- [ ] I have marked all applicable categories:
    + [ ] exception-raising bug
    + [X] RL algorithm bug
    + [ ] documentation request (i.e. "X is missing from the documentation.")
    + [ ] new feature request
- [X] I have visited the [source website](https://github.com/thu-ml/tianshou/)
- [X] I have searched through the [issue tracker](https://github.com/thu-ml/tianshou/issues) for duplicates
- [ ] I have mentioned version numbers, operating system and environment, where applicable

I have submitted a similar issue in Stable-Baselines 3: https://github.com/DLR-RM/stable-baselines3/issues/802.

In https://github.com/thu-ml/tianshou/blob/master/tianshou/policy/modelfree/sac.py, line 177:
```python
alpha_loss = -(self._log_alpha * log_prob).mean()
```
The leading coefficient is `self._log_alpha`, but in equation (18) in https://arxiv.org/pdf/1812.05905.pdf, it is alpha that is present in the loss, not log(alpha).  
Similarly, in the implementation of the authors (https://github.com/rail-berkeley/softlearning/blob/master/softlearning/algorithms/sac.py
, line 256): 
```python
alpha_losses = -1.0 * (self._alpha * tf.stop_gradient(log_pis + self._target_entropy))
```
So shouldn't line 177 of sac.py be modified accordingly?

The same potential mistake is present in the Stable-Baselines 3 implementation (https://github.com/DLR-RM/stable-baselines3/blob/master/stable_baselines3/sac/sac.py, line 215), so I wonder: is there actually a reason for this difference? In any case, I guess that it has a very limited impact on the algorithm.

**Comment by perrin-isir** [comment]
[DUPLICATE]
This has actually been discussed several times:
https://github.com/DLR-RM/stable-baselines3/issues/36
https://github.com/DLR-RM/stable-baselines3/issues/712
https://github.com/rail-berkeley/softlearning/issues/37
https://github.com/rail-berkeley/softlearning/issues/136#issuecomment-620533436

**Comment by Trinkle23897** [comment]
#258

**Comment by perrin-isir** [comment]
Thanks!
